### DISCUSSION TITLE
Safari Refused to connect to wss due to missing Content-Security-Policy (CSP)

### DISCUSSION THREAD

**Discussion by dennisat** [socketio/socket.io#4269] [Q&A]
_Only_ Safari is complaining about the [CSP](https://en.wikipedia.org/wiki/Content_Security_Policy). The consoled error is:

```
Refused to connect to wss://example.com/socket.io/?EIO=4&transport=websocket because it appears in neither the connect-src directive nor the default-src directive of the Content Security Policy.
```

Tried add the`<meta http-equiv="Content-Security-Policy"  content="connect-src https: wss:;">` but still something is missing. 

Any suggestion?

**Comment by rajathongal-beomelo** [comment]
This issue still persists
